app/views/events/_event.html.erb

```diff
@@ -6,10 +6,39 @@
       <div class="time-ago"><%= time_ago_in_words(event.created_at) %></div>
     </div>
   </div>
-  <div class="map center-block">
-    <%= map_image(event) %>
-  </div>
 
+  <% if event.picture.exists? %>
```

Comment:
you're checking if event.picture.exists? here no need to check again in the controller